Add upgrade user action to gsctl and extend upgrade guide (#322)
giantswarm/docs: src/content/reference/gsctl/upgrade-cluster.md

+++
title = "gsctl Command Reference: upgrade cluster"
description = "Detailed documentation on how to upgrade a cluster using the 'upgrade cluster' command in gsctl."
date = "2019-06-11"
type = "page"
weight = 30
+++

# `gsctl upgrade cluster`

> In order to upgrade the cluster to the next suitable version, use this syntax:

Reviewer: ```suggestion
In order to upgrade a cluster to the next suitable version, use this command:
```

```nohighlight
$ gsctl upgrade cluster <cluster-id>
```

> A message will be output showing the version to be upgraded, and a list of changelogs for the components that will be upgraded. Before confirming this action ensure you know the [upgrade process](https://docs.giantswarm.io/reference/cluster-upgrades/)] and the impact it entails.

Reviewer: <3

> There will appear a message showing the version to be upgraded and a list of changelogs for the components that will be upgraded. Before confirming the action ensure you know the [upgrade process](https://docs.giantswarm.io/reference/cluster-upgrades/)] and the impact it entails.

Reviewer: ```suggestion
A message will be output showing the version to be upgraded, and a list of changelogs for the components that will be upgraded. Before confirming this action ensure you know the [upgrade process](https://docs.giantswarm.io/reference/cluster-upgrades/)] and the impact it entails.
```

OK this was my point from earlier! :)
